tests/test_pvl.py

```diff
@@ -592,3 +594,10 @@ def test_cube_label():
     assert label['IsisCube']['Core']['Pixels']['ByteOrder'] == 'Lsb'
     assert label['IsisCube']['Core']['Pixels']['Base'] == 0.0
     assert label['IsisCube']['Core']['Pixels']['Multiplier'] == 1.0
+
+def test_load_all_sample_labels():
+    files = glob.glob(os.path.join(PDS_DATA_DIR, "*.lbl"))
+    for file in files:
+        with open(file, 'r') as f:
```

Comment:
It's failing in python3 because it expects you to open it the file in binary mode (ie open(file, 'rb')). You can also just pass pvl.load the filename and it will open it for you.